### DISCUSSION TITLE
App OAuth

### DISCUSSION THREAD

**Discussion by ohuu** [saleor/saleor#10926] [Q&A]
I'm trying to develop a simple Saleor app which responds to several Saleor webhooks but I'm a bit confused about how to validate the webhook payloads.

Specifically I'm confused by the `/.well-known/jwks.json` stuff. We're self-hosting Saleor on DigitalOcean and I'm not sure what I need to do in order to create and host this file.

I understand that `jwks.json` has something to do with OAuth but I've never used OAuth before. How do I create this file? Do I need an OAuth provider or something?

**Comment by peelar** [comment]
Hello @ohuu 👋 

Are you using [saleor-app-template](https://github.com/saleor/saleor-app-template)? It solves some of the common pains of developing an app.
